coalib/results/Diff.py

```diff
@@ -2,14 +2,24 @@
 import difflib
 import logging
 
+from pkg_resources import get_distribution
 from unidiff import PatchSet
 
+from clang.cindex import LibclangError
 from coalib.results.LineDiff import LineDiff, ConflictError
 from coalib.results.SourceRange import SourceRange
 from coalib.results.TextRange import TextRange
 from coala_utils.decorators import enforce_signature, generate_eq
 
 
+def ClangVersionCheck():
+    if get_distribution('libclang-py3').version != '3.4.0':
+        raise LibclangError('coala requires clang 3.4')
+
+
+ClangVersionCheck()
```

Comment:
you can inline it, so just put here what's inside the `ClangVersionCheck()` function (which doesn't meet btw our naming scheme ;) --> `def check_clang_version`)

Reply:
Ah you did it to test things easier... but should be possible I think to test it anyway even without a functino, there should be infos out there how to do it for such cases